### PR TITLE
Adding 18F colors

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Same as https://github.com/cfpb/cf-core/pull/52, this pulls in the 18F colors and replaces the terrible pinks and purples.

### CODE REVIEW

**Comment by ascott1** [comment]
:+1: from me

File: src/cf-typography.less [hunk 1]
```diff
@@ -48,42 +48,42 @@
 // Color variables
```
Comment (Scotchester) [inline src/cf-typography.less:48]: Same comment as the other PR about having a line up here mentioning what the `$color-` comments are referring to.

File: src/cf-typography.less [hunk 2]
```diff
@@ -48,42 +48,42 @@
...
 
 // .pull-quote
-@pull-quote_body:               #895983;
-@pull-quote_citation:           lighten(#895983, 15%);
+@pull-quote_body:               #494440; // $color-gray-warm-dark
+@pull-quote_citation:           #aeb0b5; // $color-gray-light
```
Comment (Scotchester) [inline src/cf-typography.less]: This color isn't WCAG compliant.

**Comment by Scotchester** [comment]
:+1: After addressing the one inaccessible combination.

**Comment by cfarm** [comment]
pinks and purples forever.

image

File: src/cf-typography.less (hunk 2 above)
Comment (KimberlyMunoz) [inline src/cf-typography.less]: Fixed in 0d0d351 with a color that's better looking anyway.

**Comment by KimberlyMunoz** [comment]
Thanks @Scotchester. Ready for another review. This one should pull in the cf-core colors as well to get rid of the pink links!

**Comment by Scotchester** [comment]
Thanks! :+1:

You'll need to tag and release cf-core for cf-typography to pull that stuff in, though. And cf-typography will needs its own release.

**Comment by jimmynotjim** [comment]
@KimberlyMunoz this still needs a release

**Comment by KimberlyMunoz** [comment]
Thanks for the reminder. Let me do https://github.com/cfpb/cf-core/pull/54 first so I can pull in those colors here.

**Pull request merged**